### PR TITLE
Enable refresh_token grant

### PR DESCRIPTION
**Pull request by axuan25**
Asana: https://app.asana.com/0/1128361015731305/1132216572973271

* enable token refresh using `ReachFive.refreshAccessToken`
* some packaging chores

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -57,7 +57,6 @@ dependencies {
     implementation project(":sdk-google")
     implementation project(":sdk-webview")
 
-    implementation "com.google.firebase:firebase-core:16.0.9"
```
Comment (Wadjetz) [inline app/build.gradle]: si tu le supprime, assure toi que le provider native google fonctionne

File: app/build.gradle [hunk 2]
```diff
@@ -69,5 +68,5 @@ dependencies {
     androidTestImplementation 'androidx.test:core-ktx:1.2.0'
```
Comment (roxanemace) [inline app/build.gradle]: Could you replace the single quotes by double quotes?

File: app/src/main/java/com/reach5/identity/sdk/demo/JavaMainActivity.java [hunk 3]
```diff
@@ -13,9 +13,9 @@
 import com.reach5.identity.sdk.core.JavaReachFive;
 import com.reach5.identity.sdk.core.Provider;
 import com.reach5.identity.sdk.core.models.AuthToken;
-import com.reach5.identity.sdk.core.models.SdkConfig;
+import com.reach5.identity.sdk.core.SdkConfig;
 import com.reach5.identity.sdk.core.models.OpenIdUser;
-import com.reach5.identity.sdk.core.models.requests.ProfileSignupRequest;
+import com.reach5.identity.sdk.core.models.ProfileSignupRequest;
```
Comment (roxanemace) [inline app/src/main/java/com/reach5/identity/sdk/demo/JavaMainActivity.java]: I would have preferred that you do a specific PR for your refactoring. It has nothing to do with the refresh token and it implies breaking changes for the customers. You will need to update the documentation and to list all the changes in the changelog file.

File: sdk-core/src/main/java/com/reach5/identity/sdk/core/models/requests/AuthCodeRequest.kt [hunk 4]
```diff
@@ -9,10 +9,11 @@ data class AuthCodeRequest(
...
-    val grantType: String,
     @SerializedName("redirect_uri")
     val redirectUri: String,
     @SerializedName("code_verifier")
     val codeVerifier: String
-) : Parcelable
\ No newline at end of file
+) : Parcelable {
+    @SerializedName("grant_type")
+    val grantType: String = "authorization_code"
```
Comment (roxanemace) [inline sdk-core/src/main/java/com/reach5/identity/sdk/core/models/requests/AuthCodeRequest.kt]: This parameter is never used.

**Pull request merged**